app/controllers/api/scores_controller.rb

```diff
@@ -5,7 +5,19 @@ class ScoresController < ApplicationController
     before_action :validate_score_user_id, only: :destroy
 
     def user_feed
-      scores = Score.all.order(played_at: :desc, id: :desc)
+      scores = Score.all.order(played_at: :desc, id: :desc).includes(:user)
```

Comment:
We've already added `.includes(:user)` for optimization in other PR, right? No need to add it here as it would create merge conflicts.